src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CicCase.java

```diff
@@ -87,54 +84,6 @@ public class CicCase {
     @CCD(access = {DefaultAccess.class, CaseworkerWithCAAAccess.class})
     private ComponentLauncher linkedCasesComponentLauncher;
 
-    @CCD(access = {DefaultAccess.class, CaseworkerWithCAAAccess.class},
-        typeOverride = Collection,
-        typeParameterOverride = "Flags")
-    private List<ListValue<Flags>> appellantFlags;
-
-    @CCD(access = {DefaultAccess.class, CaseworkerWithCAAAccess.class},
-        typeOverride = Collection,
-        typeParameterOverride = "Flags")
-    private List<ListValue<Flags>> caseFlags;
```

Comment:
has this field been moved or removed altogether? If removed altogether, since it was in master previously moving it to retired fields makes the most sense

Reply:
if moved, a migration is required due to cicCase prefix

Reply:
Added the fields that are removed from CicCase to RetiredFields.